development/checker/unique-jobs-name/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/config"
+
+	"os"
+)
+
+type options struct {
+	prowConfig   string
+	jobConfigDir string
+}
+
+type report struct {
+	active   bool
```

Comment:
consider renaming `active` to `foundDuplicates`

Reply:
👍